torch/csrc/jit/passes/onnx/helper.h

```diff
@@ -3,11 +3,15 @@
 #include <torch/csrc/jit/api/module.h>
 #include <torch/csrc/jit/ir/ir.h>
 
-#include <memory>
+namespace ONNX_NAMESPACE {
+enum TensorProto_DataType : int;
```

Comment:
This enum is defined from a onnx.proto, isn't it? protobuf does not guarantee stable mapping into C++ types (namely older versions of protobuf would not specify the type)